pkg/minishift/addon/parser/addon_parser_test.go

```diff
@@ -277,3 +277,18 @@ func Test_multiple_addon_definitions_create_error(t *testing.T) {
 
 	assert.Nil(t, addon, "No addon should have been returned")
 }
+
+func TestValidAddonDirAndName(t *testing.T) {
```

Comment:
why we even need this test? Since add-on test is already in place which anyway going to use the same parse and also validate the directory mismatch. Let's remove this test.